frontend-react/yarn.lock

```diff
@@ -1878,27 +1878,6 @@
     slash "^3.0.0"
     write-file-atomic "^4.0.2"
 
-"@jest/transform@^29.6.1":
```

Comment:
I'm not entirely sure why this is being removed but I do see we still have the `"@jest/transform@^29.3.1", "@jest/transform@^29.5.0", "@jest/transform@^29.6.1"` dependency above so maybe it's an unused version 🤔

Reply:
Probably when merging from master.